examples/classification/eval_imagenet.py

```diff
@@ -44,8 +44,12 @@ def main():
     label_names = directory_parsing_label_names(args.val)
     n_class = len(label_names)
     iterator = iterators.MultiprocessIterator(
-        dataset, args.batchsize, repeat=False, shuffle=False,
-        n_processes=6, shared_mem=300000000)
+        dataset,
+        args.batchsize,
+        repeat=False,
+        shuffle=False,
+        n_processes=6,
+        shared_mem=300000000)
```

Comment:
it looks no modification.
do we really need this change?

Reply:
I applied yapf. Should I make it back to before?

Reply:
yes, please make it back.
it does not exceed any line length limitation

Reply:
OK